src/test/java/com/google/api/codegen/testdata/python_test_library.baseline

```diff
@@ -51,6 +57,23 @@ class TestLibraryServiceClient(unittest2.TestCase):
 
         grpc_stub.CreateShelf.assert_called_once()
 
+    @mock.patch('google.gax.config.API_ERRORS', (CustomException,))
+    @mock.patch('google.gax.config.create_stub')
+    def test_create_shelf_exception(self, mock_create_stub):
+        # Mock gRPC layer
+        grpc_stub = mock.Mock()
+        mock_create_stub.return_value = grpc_stub
+
+        client = library_service_client.LibraryServiceClient()
+
+        # Mock request
+        shelf = library_pb2.Shelf()
+
+        # Mock exception response
+        grpc_stub.CreateShelf.side_effect = CustomException()
+
+        self.assertRaises(errors.GaxError, client.create_shelf, shelf)
```

Comment:
@lukesneeringer Fair enough, we can revisit this part when we settle the GaxError design.